ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -6,13 +6,6 @@
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "CRU"
   },
-  {
```

Comment:
you've removed perms here. Please add back. It's easier for me to add just C and U when we toggle on

Reply:
@Gogetter please add these back